runner.sh

```diff
@@ -116,6 +119,7 @@ function print_usage {
     echo "    --no-run          : don't run notebooks"
     echo "    --no-checks       : don't run code checks"
     echo "    --autofix         : autofix where possible"
+	echo "    --copyright       : check whether every source code and notebook has a copyright header"
```

Comment:
there are some indentation misalignments in the script

Reply:
Fixed the issue to make the indentation consistent in the function, plus a few other places in my previous commit. Thanks @wyli !